src/Components/JournalPrompt/JournalPrompt.js

```diff
@@ -1,12 +1,28 @@
-import React from "react";
+import React, {useState} from "react";
+import { Link } from "react-router-dom";
+import './JournalPrompt.css';
+
+const JournalPrompt = ( {updateJournal} ) => {
+  const [newEntry, setNewEntry] = useState('');
+
+  const updateEntry = () => {
+    const input = document.getElementById('journal-entry');
+    setNewEntry(input.value);
+  }
+
+  const submitEntry = () => {
+    updateJournal(newEntry);
+  }
 
-const JournalPrompt = () => {
   return (
-    <form>
-      <h2>Prompt</h2>
-      <input type="text"/>
-    </form>
+    <div className="prompt-container">
+      <form>
+        <h2 className="prompt">Why are you feeling this way?</h2>
+        <textarea id="journal-entry" onChange={updateEntry}></textarea>
+        <Link to="/how-you-felt" className="next-button" onClick={submitEntry}>→</Link>
```

Comment:
nice dude, this looks great! Maybe an extension, but I was clicking through and I wanted to see some sort of message that thanks or congratulates the guest for journaling! And maybe says "we hope to see you tomorrow?" I was missing that serotonin hit

Reply:
love that idea @chrissycooper ^^